inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -468,4 +470,38 @@ public function update_safelist_items( $old_version ) {
 
 		update_option( 'wp_rocket_settings', $options );
 	}
+
+	/**
+	 * Display a notification on wrong license.
+	 *
+	 * @return void
+	 */
+	public function display_wrong_license_notice() {
+		if ( ! $this->can_display_notice( false ) ) {
+			return;
+		}
+
+		$boxes = get_user_meta( get_current_user_id(), 'rocket_boxes', true );
+
+		if ( in_array( 'rucss_wrong_licence_notice', (array) $boxes, true ) ) {
+			return;
+		}
+
+		$message = sprintf(
+		// translators: %1$s = plugin name, %2$s = opening anchor tag, %3$s = closing anchor tag.
+			__( "%1\$s: <p>We couldn't generate the used CSS because you're using a nulled version of WP Rocket. You need an active license to use the Remove Unused CSS feature and further improve your website's performance..</p>%2\$sClick here to get a WP Rocket single license at 10%% off!%3\$s", 'rocket' ),
```

Comment:
Avoid adding HTML tags inside the translatable string. If necessary you can split the string into multiple parts.